Kinvey/Kinvey/MIC.swift

```diff
@@ -217,13 +216,19 @@ public enum MICUserInterface {
     
     /// Uses SFSafariViewController
     case safari
+
+    /// Uses SFAuthenticationSession
+    case safariAuthenticationSession
```

Comment:
@heyzooi is there a reason for not making this the default?

Reply:
well, not really since if you not running in iOS 11 we just use to the current default, which is the Safari View Controller. do you want me to make this the default?

Reply:
From my understanding, it seems to make sense to use the authSession all the time (when available).

Reply:
sure, let me do that and also try to mock some data for unit tests

Reply:
write unit tests for this is basically impossible now since there's no way to accept the alert displayed by the system. I tried many ways to accomplish that, KIF, XCUIApplication, UIWindow hierarchy, nothing worked!